app/controllers/concerns/decidim/reporting_proposals/proposals_controller_override.rb

```diff
@@ -49,8 +49,19 @@ def complete
           redirect_to "#{Decidim::ResourceLocatorPresenter.new(@proposal).path}/preview" if reporting_proposal?
         end
 
+        def edit_form
+          form_attachment_model = form(AttachmentForm).from_model(@proposal.attachments.first)
+          @form = form_reporting_proposals_model
+          @form.attachment = form_attachment_model
+          @form
+        end
+
```

Comment:
```suggestion
```
I think we can skip this override if we rename  `form_reporting_proposals_model` to `form_proposal_model`

Reply:
If I don't override this method, then has_no_image will be undefined, because has_no_image is not in Decidim::Proposals::ProposalForm

Reply:
Of course. The component check is in the method `form_proposal_model`. 
I may not add a new method, but use a method with the same name as the one already defined in `edit_form`.

Changed